Toolset/libraries/revcommonlibrary.livecodescript

```diff
@@ -1352,6 +1354,14 @@ function revFontGetSystemFont
          return "Helvetica"
          break
          
+      case "iphone"
```

Comment:
Hmm... I wonder if this should just return `(Default)` on all platforms now.

Reply:
perhaps we need a way to query the font name and size for the themed fonts

Reply:
Is that something that would need to be added to the engine?

Reply:
yes it is

Reply:
I just did a search for "revFontGetSystemFont" and the only hits are in the common library.  I also used a tool that has produced hits within binary stacks before that also came up empty.  I also searched for "revFontApplySystemStyle" (only place the above function is used) and it is only used in 2 files:  revaboutcardbehavior.livecodescript (1) and revonlinelibrary.rev (4).  From that, it seems that this is more of an IDE thing and not needed/used in stand alone apps.

Reply:
Hmm... it could be an undocumented handler that is still used by some users... I'm not sure